unstructured/ingest/cli/cmds/databricks_volumes.py

```diff
@@ -0,0 +1,65 @@
+import typing as t
+from dataclasses import dataclass
+
+import click
+
+from unstructured.ingest.cli.interfaces import CliConfig
+from unstructured.ingest.connector.databricks_volumes import (
+    DatabricksVolumesWriteConfig,
+    SimpleDatabricksVolumesConfig,
+)
+
+CMD_NAME = "databricks-volumes"
+
+
+@dataclass
+class DatabricksVolumesCliConfig(SimpleDatabricksVolumesConfig, CliConfig):
+    @staticmethod
+    def get_cli_options() -> t.List[click.Option]:
+        options = [
+            click.Option(["--host"], type=str, default=None),
+            click.Option(["--account-id"], type=str, default=None),
+            click.Option(["--username"], type=str, default=None),
+            click.Option(["--password"], type=str, default=None),
+            click.Option(["--client-id"], type=str, default=None),
+            click.Option(["--client-secret"], type=str, default=None),
+            click.Option(["--token"], type=str, default=None),
+            click.Option(["--profile"], type=str, default=None),
+            click.Option(["--azure-workspace-resource-id"], type=str, default=None),
+            click.Option(["--azure-client-secret"], type=str, default=None),
+            click.Option(["--azure-client-id"], type=str, default=None),
+            click.Option(["--azure-tenant-id"], type=str, default=None),
+            click.Option(["--azure-environment"], type=str, default=None),
+            click.Option(["--auth-type"], type=str, default=None),
+            click.Option(["--cluster-id"], type=str, default=None),
+            click.Option(["--google-credentials"], type=str, default=None),
+            click.Option(["--google-service-account"], type=str, default=None),
```

Comment:
can you add help/description text for all of these? these are a lot of options, it would help for a user to have some context.